src/thebelab.js

```diff
@@ -436,6 +469,21 @@ export function requestBinder({
           break;
         case "ready":
           es.close();
+          try {
+            // save the current connection url+token to reuse later
+            window.localStorage.setItem("thebe_binder_pod_url_" + url, msg.url);
```

Comment:
maybe a single item with the two fields as JSON? e.g. info 


```suggestion
            window.localStorage.setItem("thebe-binder-" + url, JSON.stringify({url: msg.url, token: msg.token, started: new Date()}));
```

then `started` can also be used to ignore saved data older than e.g. one day